code/modules/antagonists/changeling/powers/shriek.dm

```diff
@@ -41,7 +41,9 @@
 /datum/action/changeling/dissonant_shriek/sting_action(mob/user)
 	..()
 	for(var/obj/machinery/light/L in range(5, usr))
-		L.on = 1
+		L.on = TRUE
 		L.break_light_tube()
+		stoplag()
```

Comment:
Is it possible we can move the `empulse` currently after this loop before it instead? Just on the basis of how important the EMP is to the dissonant shriek, should happen immediately.

Reply:
I'll hit that tomorrow!